scripts-lib.sh

```diff
@@ -1,25 +1,28 @@
 # Common functions for %post or %pre kickstart section
 
-# is_rhel8 KSTESTS_OS_NAME KSTESTS_OS_VERSION
-# Are we testing rhel 8 (based on the name and version detected from boot.iso) ?
-function is_rhel8 {
+# get_platform KSTEST_OS_NAME KSTEST_OS_VERSION
+# Get the platform (fragments platform / --platform launch script option).
+# For fedora return only "fedora" prefix (only fedora_rawhide platform is supported currently).
+function get_platform {
    local os_name=$1
    local os_version=$2
-   if [ ${os_name} == "rhel" ] && [ ${os_version:0:1} == "8" ] ; then
-       return 0
-   else
-       return 1
-   fi
-}
+   local platform=""
 
-# is_rhel9 KSTESTS_OS_NAME KSTESTS_OS_VERSION
-# Are we testing rhel 9 (based on the name and version detected from boot.iso) ?
-function is_rhel9 {
-   local os_name=$1
-   local os_version=$2
-   if [ ${os_name} == "rhel" ] && [ ${os_version:0:1} == "9" ] ; then
-       return 0
-   else
-       return 1
+   if [ ${os_name} == "rhel" ] || [ ${os_name} == "centos" ] ; then
+       platform=rhel
+       if [ ${os_version:0:1} == "8" ] ; then
+           platform=rhel8
+       fi
+       if [ ${os_version:0:1} == "9" ] ; then
+           platform=rhel9
+       fi
+       if [ ${os_version:0:2} == "10" ] ; then
+           platform=rhel10
+       fi
```

Comment:
I think it would make sense to make the function more general, so that it doesn't have to be updated in the future for RHEL-11,... For example:
```
os_major=$(echo "$os_version" | grep -oE "^[[:digit:]]+")
if [ -n "$os_major" ] ; then
    platform="rhel${os_major}"
fi
```

Reply:
Updated